src/components/Header.tsx

```diff
@@ -21,9 +35,16 @@ function Header () {
             <li className="nav-item navs">
               <a className="nav-link" href="/contact">Contact</a>
             </li>
-            <li className="nav-item navs">
-              <a className="nav-link" href="/join">Register/Login</a>
-            </li>
+            {
+              fire.auth().currentUser != null
+                ? <li className="nav-item navs">
+                  <button onClick={() => logout()} className="nav-link logout">Logout</button>
+                </li>
+                : <li className="nav-item navs">
+                  <a className="nav-link" href="/join">Join</a>
```

Comment:
```suggestion
                  <Link className="nav-link" to="/join">Join</a>
```
can you replace all `a` tags with `Link` tags

Reply:
Done